.gitignore

```diff
@@ -1,3 +1,7 @@
+##DjANGO
 *.pyc
 *.sqlite
-media/
\ No newline at end of file
+media/
```

Comment:
Can't have the media folder in repository. It will cause all the documents uploaded on the website to be pushed into the reppo